src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  AppBar,
+  Toolbar,
+  Typography,
+  IconButton,
+  Button,
+  Menu,
+  MenuItem,
+} from '@mui/material';
+import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
+import SearchIcon from '@mui/icons-material/Search';
+import MenuIcon from '@mui/icons-material/Menu';
+import { InputWithIcon } from '../InputWithIcon/InputWithIcon';
+
+const dummyCategories = ['HandBags', 'Watches', 'Skincare', 'Jewllery', 'Apparels'];
+
+// eslint-disable-next-line react/prop-types, no-unused-vars
```

Comment:
why would you disable eslint here?

Reply:
I will create the prop-Types.
But I will disable {no-used-vars} for now until we connect with the backend